lightweight/common/eventdata.cpp

```diff
@@ -4,7 +4,7 @@ QDBusArgument &operator<<(QDBusArgument &argument, const EventData &eventData)
 {
     argument.beginStructure();
     argument << eventData.calendarUid
-             << eventData.uniqueId
+             << eventData.instanceId
```

Comment:
Good catch. I would vote to remove it completely from the lightweight interface.
`grep -lr recurrenceId /usr/lib64/qt5/qml/` is not returning any match and `grep -lr recurrenceId /usr/share` returns a single match in the DBus calendar interface, which is expected. There is another match in /usr/share/jolla-calendar/pages/EventEditPage.qml in a test to know if the event is an exception.

So as far as I know with this information, it is safe to remove it from the lightweight interface. As you mentioned, one could add the incidenceUid instead. But removing the recurrenceId somehow hides an implementation detail that is not used. If we want to expose the parent of an exception at one moment, it would be better to send the instanceId of the parent instead.

Reply:
Removal is fine here too. Either way.

Reply:
I've removed it, it makes things simpler. We can still add it later if needed, like we did for the cancel status for instance.